src/category_theory/limits/cones.lean

```diff
@@ -178,6 +176,20 @@ namespace cones
 { hom := { hom := φ.hom },
   inv := { hom := φ.inv, w' := λ j, φ.inv_comp_eq.mpr (w j) } }
 
+def postcompose {G : J ⥤ C} (α : F ⟶ G) : cone F ⥤ cone G :=
+{ obj := λ c, { X := c.X, π := c.π ⊟ α },
+  map := λ c₁ c₂ f, { hom := f.hom, w' :=
+  by intro; erw ← category.assoc; simp [-category.assoc] } }
```

Comment:
either in the previous line or indent by 2 more spaces, otherwise it looks like its an additional structure field.